### PR TITLE
sync rep: new master election

### PR DESCRIPTION
**Pull request by jeroiraz**
Signed-off-by: Jeronimo Irazabal <jeronimo.irazabal@gmail.com>

### CODE REVIEW

File: src/code-examples/go/connect-healthcheck/go.mod [hunk 1]
```diff
@@ -2,7 +2,7 @@ module database-create
 
 go 1.18
 
-require github.com/codenotary/immudb v1.3.1
+require github.com/codenotary/immudb v1.4.0
```
Comment (byo) [inline src/code-examples/go/connect-healthcheck/go.mod:5]: I'm not sure if this is needed - this sample, if used in 1.4.0 docs, will also be checked against 1.4.0 version.

File: src/master/connecting/sdks.md [hunk 2]
```diff
@@ -16,11 +16,11 @@ The most well-known and recommended immudb SDK is written in [Golang](https://go
...
-| `JAVA`               | immudb team  | 1.3.2       |   [link](https://github.com/codenotary/immudb4j)  |                                      |
+| `go`               | immudb team  | 1.4.0       |     [link](https://pkg.go.dev/github.com/codenotary/immudb/pkg/client)  |                                   |
+| `python`               | immudb team  | 1.4.0       |  [link](https://github.com/codenotary/immudb-py) |                                     |
+| `JAVA`               | immudb team  | 1.4.0       |   [link](https://github.com/codenotary/immudb4j)  |                                      |
 | `NODE`               | immudb team | 1.2.1       |   [link](https://github.com/codenotary/immudb-node) | Verification is not working                 |
-| `ruby`               | Community ([Ankane](https://github.com/ankane))  | 1.3.2       |   [link](https://github.com/ankane/immudb-ruby) |                 |
+| `ruby`               | Community ([Ankane](https://github.com/ankane))  | 1.4.0       |   [link](https://github.com/ankane/immudb-ruby) |                 |
```
Comment (byo) [inline src/master/connecting/sdks.md:23]: Should we claim that it is supported before ruby SDK is updated ?

Honestly I think that this table is not correct, it should contain something like `most recent SDK version` and `minimum required immudb version`

File: src/code-examples/go/connect-healthcheck/go.mod (hunk 1 above)
Comment (jeroiraz) [reply src/code-examples/go/connect-healthcheck/go.mod:5]: I think it's better to use the latest go sdk in the samples, even if we check them using older immudb servers

Probably we need to have version the samples as well

File: src/master/connecting/sdks.md (hunk 2 above)
Comment (jeroiraz) [reply src/master/connecting/sdks.md:23]: It is even possible that we may need a table for each SDK, listing the immudb versions that work with that SDK version

**Pull request merged**